pkg/test/resourcefixture/testdata/basic/bigqueryanalyticshub/v1alpha1/bigqueryanalyticshublisting-full/dependencies.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: bigquery.cnrm.cloud.google.com/v1beta1
+kind: BigQueryDataset
+metadata:
+  name: bigquerydataset${uniqueId}
+spec:
+  friendlyName: my-bigquerydataset
+---
+apiVersion: bigqueryanalyticshub.cnrm.cloud.google.com/v1alpha1
+kind: BigQueryAnalyticsHubDataExchange
+metadata:
+  name: bigqueryanalyticshubdataexchange${uniqueId}
+spec:
+  displayName: my_data_exchange
+  location: US
+  projectRef:
+    external: ${projectId}
+  resourceID: bigqueryanalyticshubdataexchange${uniqueId}
```

Comment:
Maybe not related, But I am not sure why we need to set resourceID when the metadata.name has the same value?